SETUP.md

```diff
@@ -107,7 +107,7 @@ Then, create the file `$RECO_ENV/etc/conda/activate.d/env_vars.sh` and add:
     export PYSPARK_PYTHON=$RECO_ENV/bin/python
     export PYSPARK_DRIVER_PYTHON=$RECO_ENV/bin/python
     export SPARK_HOME_BACKUP=$SPARK_HOME
-    unset SPARK_HOME
+    export SPARK_HOME=/dsvm/tools/spark/current
```

Comment:
This works only for DSVM, not in general for Linux and MacOS.

Reply:
yes, that's right. We need to change this because unset creates a problem. So we can have this path, or add a dummy one, or maybe add a comment here saying that please change the path to your current SPARK_HOME...

Reply:
Well, this path is the standard SPARK_HOME that DSVM is deployed with. So why do you need to do this line, unless someone modified SPARK_HOME after deploying the DSVM?

Reply:
In the dsvm `SPARK_HOME` is not defined, at least in two VMs I saw

Reply:
I see. It works on the NC6_Promo though.